staticaddr/loopin/manager.go

```diff
@@ -562,8 +606,33 @@ func (m *Manager) initiateLoopIn(ctx context.Context,
 	}
 	totalDepositAmount := tmp.TotalDepositAmount()
 
+	// If the selected amount would leave a dust change output or exceeds
+	// the total deposits value, we return an error.
+	dustLimit := lnwallet.DustLimitForSize(input.P2TRSize)
```

Comment:
nit: why not use SwapAmountFromSelectedAmount?

Reply:
Sorry, what do you mean by SwapAmountFromSelectedAmount? I can't follow.